sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -469,6 +472,10 @@ object Radar {
         this.context = context.applicationContext
         this.handler = Handler(this.context.mainLooper)
 
+        if (context is Activity) {
+            this.activity = context
+        }
```

Comment:
interesting — why do we do this?

Reply:
We need some casting here or things get weird later in terms of types when we try and call it later in the sdk. Essentially its to handle all the sub-classing android has.